### ISSUE TITLE
Module Options

### ISSUE THREAD

**Issue by asbisen** [sglyon/PlotlyJS.jl#108]
How can we incorporate `module.exports` in `PlotlyJS.jl` plots as defined [here](https://github.com/plotly/plotly.js/blob/master/src/plot_api/plot_config.js#L22-L104). This would allow creating static plots and managing interactions with the plot.

**Comment by sglyon** [comment]
That's a great idea.

I have hardcoded `showLink: false` for each plot ([see here](https://github.com/sglyon/PlotlyJS.jl/blob/6901868ff51bf0026280bff1616fddd451b72d1d/src/display.jl#L21)), but I don't have an entry point for the other options.

I don't have time to do this myself right now, but if you are up for submitting a pull request I'd be happy to review it.

**Comment by sglyon** [comment]
I'd be happy to accept pull requests that implement this.

For a current list of such options, see here https://plot.ly/javascript/configuration-options/

**Comment by sglyon** [comment]
included in #208

**Comment by sglyon** [comment]
now included!

For now, please use the master branch

```julia
] # enter pkg mode
(v1.0) pkg> checkout PlotlyJS
```

If you want save figures you can do

```
] # enter package mode
(v1.0) pkg> add https://github.com/sglyon/ORCA.jl.git#master
```

If this doesn't resolve your issue here, please reopen the issue and leave a comment describing what went wrong
